themes/default/content/pulumi-up/_index.md

```diff
@@ -37,6 +37,53 @@ event_sections:
         of Pulumi customers about how they have adopted Cloud Engineering best practices and how those practices have
         helped them be successful in this new era of software development.
 
+workshops:
+    - title: Introduction To Pulumi
+      url: /resources/introduction-to-pulumi/?date=2021/04/20
+      datetime: 2021-04-20T09:00:00-07:00
+      description: |
+        In this workshop, you will learn the fundamentals of Infrastructure as Code through guided exercises using Pulumi's Cloud Engineering platform.
+
+    - title: Getting Started with Automation API
+      url: /resources/getting-started-with-automation-api
+      datetime: 2021-04-21T09:00:00-07:00
+      description: |
+        Pulumi’s Automation API opens new horizons for infrastructure provisioning. In this workshop, we’ll examine the powerful new capabilities of Pulumi’s latest feature by building a Python & Flask web application that lets developers deploy applications at the click of a button.
```

Comment:
We should use `you'll` over `we'll` almost every time. _We_ aren't exploring Pulumi's capabilities, _they_ are.

```suggestion
        Pulumi’s Automation API opens new horizons for infrastructure provisioning. In this workshop, you’ll examine the powerful new capabilities of Pulumi’s latest feature by building a Python & Flask web application that lets developers deploy applications at the click of a button.
```